sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -700,10 +700,12 @@ public virtual Response<T> GetEntity<T>(string partitionKey, string rowKey, IEnu
         /// <param name="rowKey">The rowKey that identifies the table entity.</param>
         /// <param name="select">Selects which set of entity properties to return in the result set. Pass <c>null</c> to retreive all properties.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
-        /// <returns>The <see cref="Response"/> indicating the result of the operation.</returns>
-        /// <exception cref="RequestFailedException">Exception thrown if the entity doesn't exist.</exception>
+        /// <returns> NullableResponse of T whose HasValue property will return <c>true</c> if the entity existed, otherwise <c>false</c>.</returns>
```

Comment:
```suggestion
        /// <returns> The <see cref="NullableResponse{T}" whose <c>HasValue</c> property will return <c>true</c> if the entity existed, otherwise <c>false</c>.</returns>
```

Reply:
Thanks - I couldn't remember what the generic syntax was.